shared/syncs.schema.json

```diff
@@ -19,9 +19,18 @@
         "sobjectTypePlural": {
             "type": "string"
         },
+        "formFactor": {
+            "type": "string"
+        },
         "layoutType": {
             "type": "string"
         },
+        "mode": {
```

Comment:
Same thing for mode, we could add enum with the supported values. See https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/ui_api_resources_record_layout.htm